templates/lisp/files/README.md

```diff
@@ -6,6 +6,8 @@
 </a>
 </p>
 
+This template provides the most basic configuration for running a Lisp Huchentoot web server for Platform.sh.  It can be used to build a very rudimentary application but is intended primarily as a documentation reference.  You are free to al
```

Comment:
```suggestion
This template provides the most basic configuration for running a Lisp Huchentoot web server for Platform.sh.  It can be used to build a very rudimentary application but is intended primarily as a documentation reference.  It is meant to be a starting point and can be modified to fit your own needs.
```